src/scipp/compat/xarray_compat.py

```diff
@@ -42,11 +47,50 @@ def from_xarray(obj: Union[xr.DataArray, xr.Dataset]) -> VariableLike:
         raise ValueError(f"from_xarray: cannot convert type '{type(obj)}'")
 
 
-def _var_from_xarray(xr_obj: Union[xr.Coordinate, xr.DataArray]):
+def to_xarray(obj: VariableLike) -> Union[xr.DataArray, xr.Dataset]:
+    """Convenience method to convert a scipp object into the corresponding
+    xarray object.
+
+    Parameters
+    ----------
+    obj:
+        The scipp object to convert; must be either a DataArray
+        or Dataset object.
+
+    Returns
+    -------
+    :
+        The converted xarray object.
+
+    See Also
+    --------
+    scipp.compat.from_xarray
+    """
+
+    if isinstance(obj, DataArray):
+        return to_xarray_dataarray(obj)
+    elif isinstance(obj, Dataset):
+        return to_xarray_dataset(obj)
+    else:
+        raise ValueError(f"to_xarray: cannot convert type '{type(obj)}'")
+
+
+def _var_from_xarray(xr_obj: Union[xr.Coordinate, xr.DataArray]) -> Variable:
```

Comment:
`xarray` also has something called `Variable`, it is, I think used as a base class, but I am not sure. Can you make use of that?

Reply:
The type hints. There isn't anything in the function where you could use it?

Reply:
`Variable` doesn't seem to be a base class of `DataArray`. Also, these are not new changes, so I am tempted to leave it as is.